cli/commands/flags.go

```diff
@@ -192,6 +194,18 @@ func NewGlobalFlags(opts *options.TerragruntOptions) cli.Flags {
 			EnvVar:      "TERRAGRUNT_NO_COLOR",
 			Usage:       "If specified, Terragrunt output won't contain any color.",
 		},
+		&cli.BoolFlag{
+			Name:        FlagNameTerragruntJsonLog,
+			Destination: &opts.JsonLogFormat,
+			EnvVar:      "TERRAGRUNT_JSON_LOG",
+			Usage:       "If specified, Terragrunt will outout logs in JSON format.",
+		},
+		&cli.BoolFlag{
+			Name:        FlagNameTerragruntTfLogJson,
+			Destination: &opts.TerraformLogsToJson,
+			EnvVar:      "TERRAGRUNT_TF_JSON_LOG",
+			Usage:       "If specified, Terragrunt will wrap Terraform output in JSON.",
```

Comment:
Is it wrapping both Terraform `stdout` and `stderr`? Or just `stderr`?

```suggestion
			Usage:       "If specified, Terragrunt will wrap Terraform stdout and stderr in JSON.",
```